lib/wor/paginate/formatter.rb

```diff
@@ -16,7 +16,7 @@ def format # rubocop: disable Metrics/MethodLength
           page: serialized_content,
           count: count,
```

Comment:
Should this param also been permitted to be overridden by an option?

Reply:
🤔 is hard to tell, because this only counts the result of the single page, not of the whole collection to paginate, so I don't know if it is a need also to overwrite this value (if so, you would have to pass it for every page requested). @mtejedorwolox as the author of the issue, what do you think of this?